test/test_distributions.py

```diff
@@ -722,6 +722,8 @@ def ref_log_prob(idx, x, log_prob):
             self.assertAlmostEqual(log_prob, expected, places=3)
 
         self._check_log_prob(FisherSnedecor(df1, df2), ref_log_prob)
+#        self.assertEqual(FisherSnedecor(df1, df2).entropy().data,
```

Comment:
I commented this out because the calculation for the entropy in `scipy` for F-Distribution doesn't converge. There is nothing to check with.

Reply:
However, this converges if the values `df1` and `df2` are integers.

Reply:
`Scipy`'s implementation seems to be false. The source for my implementation is specified with the source code.

Reply:
Could to add your comment to the code, to warn future testers from trying to test against the incomplete Scipy implementation? And remove the commented-out line.

Reply:
And 👍 for having a more solid implementation than scipy.